test/integration/linux/monitoring/latest/default_test.rb

```diff
@@ -0,0 +1,52 @@
+# Copyright:: 2021 Google LLC
```

Comment:
The following test files have a lot of similar code. Can we parametrize them to reduce the maintenance cost in the future?

- test/integration/linux/logging/latest/default_test.rb
- test/integration/linux/monitoring/latest/default_test.rb
- test/integration/linux/ops-agent/2.0.1/default_test.rb
- test/integration/linux/ops-agent/latest/default_test.rb

Reply:
Will open issue and refactor after we merge.

Reply:
Filed https://github.com/GoogleCloudPlatform/google-cloud-ops-agents-chef/issues/4